aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -262,11 +262,12 @@ void unary_op(const Tensor& self, const Tensor& output, std::string op_name, Una
     return;
   }
   auto input = dtype.has_value() ? self.to(dtype.value()) : self;
+  TORCH_CHECK(input.scalar_type() != ScalarType::Long, "Input type shouldn't be int64");
```

Comment:
Can you please rephrase it to `MPS does not support cumsum op with int64 input` or something similar? We have a search pattern in `test_mps.py` to skip these tests with int64 issues

Reply:
Changed the error message, also added a test case